script.js

```diff
@@ -0,0 +1,69 @@
+let developers = [
+    {
+        "name": "Carolin Bloeme",
+        "id": "carolin-bloeme"
+    },
+    {
+        "name": "Sun Jun",
+        "id": "sun-jun"
+    },
+    {
+        "name": "Song Bao",
+        "id": "song-bao"
+    },
+    {
+        "name": "Olivia Arribas",
+        "id": "olivia-arribas"
+    },
+    {
+        "name": "Samah abu laymun ",
+        "id": "samah-abu-laymun"
+    }
+]
+
+let ul = document.getElementById('developers');
+let li = "";
```

Comment:
I would suggest another name that the `li`, maybe `itemContainer`. I know that you mean it is the list element, but it is actually a div. So maybe if we keep it general would be better